tests/rabbitmq_example/rabbitmq_example.services.yml

```diff
@@ -0,0 +1,30 @@
+services:
+  rabbitmq_example_event_subscriber:
+    class: Drupal\rabbitmq\rabbitmq_example\EventSubscriber\RabbitmqSubscriber
+    tags:
+      - {name: event_subscriber}
+  rabbitmq.connection.factory:
+    class: '\Drupal\rabbitmq\Connection'
+    arguments: ['@settings', '@logger.channel.rabbitmq']
+
+  rabbitmq.config:
+    producers:
+      example:
+        connection: default
+        exchange_options:
+          name: 'dasp'
+          type: direct
+          passive: false
+          durable: true
+          auto_delete: false
+          internal: false
+          nowait: false
```

Comment:
it's the same config that uses this symfony bundle: https://github.com/videlalvaro/rabbitmqbundle and the author of that bundle works in rabbitmq organisation...
Also I test that scenario and works in that way.